server/node/README.md

```diff
@@ -7,3 +7,72 @@ Start the server by running:
 ```bash
 npm start
 ```
+
+---
+
+### `GET /paypal-api/auth/browser-safe-client-token`
+
+**Description:**  
+Generates and returns a browser-safe client token for initializing the PayPal JavaScript SDK on the client side.
+
+**Responses:**
+
+- `200 OK`  
+  Returns a JSON object with the client token.
+
+  ```json
+  {
+    "client_token": "abc123..."
+  }
+  ```
+
+- `500 Internal Server Error` returned if token generation fails.
```

Comment:
I wonder if we should remove the specifics about the error use case. There are lots of different status codes that can be returned when their is an error. Our server code just echos back whatever status code and error response the auth api or orders api returns.
```suggestion
```

Maybe we can have a general message about this and link out to the following pages:
- https://developer.paypal.com/api/rest/responses/
- https://developer.paypal.com/api/rest/reference/orders/v2/errors/